koans/DataTypesKoans.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace PhpKoans;
+
+use PHPUnit\Framework\TestCase;
+use stdClass;
+
+defined('__') or define('__', null);
+
+class DataTypesKoans extends TestCase
+{
+    /**
+     * @testdox Strings can be created using single quotes
+     */
+    public function testStringsWithSingleQuotes()
```

Comment:
I don't understand these tests, if you just put a true the test passes.
Maybe you can give it a try

Reply:
Okay I will try to improve them :)